.github/workflows/go-coverage-only.yml

```diff
@@ -0,0 +1,51 @@
+name: Go Coverage Only Report
+on:
+  push:
+    branches:
+      - master
+jobs:
+  coverage-report:
+    name: unit-tests
+    strategy:
+      matrix:
+        go-version: [1.15.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+      with:
+          persist-credentials: false
+    - name: Run Go mod tidy
+      run: go mod tidy
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Generate Coverage
+      run: |
+        go mod vendor
+        mkdir -p test-results
+        go test -mod=vendor -v ./... -count=1 -coverprofile cover.out
+    - name: CodeCov
+      run: |
+        bash <(curl -s https://codecov.io/bash)
+    - name: Check if total coverage is greater then 0
```

Comment:
you can skip this part - it doesn't has any effect

Reply:
removed irrelevant part